Mathlib/Algebra/DirectLimit.lean

```diff
@@ -347,6 +356,17 @@ theorem lift_unique [Nonempty ι] [IsDirected ι (· ≤ ·)] (F : DirectLimit G
   DirectLimit.induction_on x fun i x => by simp
 #align add_comm_group.direct_limit.lift_unique AddCommGroup.DirectLimit.lift_unique
 
+lemma lift_injective [Nonempty ι] [IsDirected ι (· ≤ ·)]
```

Comment:
While adding the injective lemma for abelian groups, I noticed that `AddCommGroup.DirectLimit.lift` is a ℤ-linear map instead of an additive group map, is this by design? `Ring.DirectLimit.lift` is a ring homomorphism though.

Reply:
[AddCommGroup.DirectLimit](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/DirectLimit.html#AddCommGroup.DirectLimit) is defined in terms of Module.DirectLimit and [AddCommGroup.DirectLimit.lift](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/DirectLimit.html#AddCommGroup.DirectLimit.lift) in terms of Module.DirectLimit.lift, so I guess the author just kept the resulting LinearMap as it's easy to get the AddMonoidHom from it using [LinearMap.toAddMonoidHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Module/LinearMap.html#LinearMap.toAddMonoidHom).

Reply:
I feel like `AddCommGroup.DirectLimit.lift` should be an `AddMonoidHom`